node/octicons/index.js

```diff
@@ -0,0 +1,60 @@
+const icons = require('./build/data.json')
+
+for (const key of Object.keys(icons)) {
+  const icon = icons[key]
+
+  // Set the symbol for easy access
+  icon.symbol = key
+
+  // Set all the default options
+  icon.options = {
+    version: '1.1',
+    width: icon.width,
+    height: icon.height,
+    viewBox: `0 0 ${icon.width} ${icon.height}`,
+    class: `octicon octicon-${key}`,
+    'aria-hidden': 'true'
+  }
+
+  // Function to return an SVG object
+  icon.toSVG = options => {
+    return `<svg ${icon.getAttributes(options)}>${icon.path}</svg>`
```

Comment:
Does this template literal produce the expected result? Looks like `getAttributes` returns a list and I think lists are joined with `","` not `" 
 "`. 
![image](https://user-images.githubusercontent.com/4608155/53303334-0c6aa200-381e-11e9-9737-90129598f5b8.png)